From 224d38249a449557714aedc2695d1a14279f5510 Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Sun, 4 Nov 2012 20:28:21 +0200
Subject: [PATCH 01/22] [#3583229]Fix for broken DnD in ExoTreeView.

The commit 441db508ebf12b6cb1849ad1a244dcd668cccf82 added the call to
gtk_tree_view_set_reorderable() which happened to be incompatible with DnD.
---
 src/gtk/fm-standard-view.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gtk/fm-standard-view.c b/src/gtk/fm-standard-view.c
index f0f791c..175cced 100644
--- a/src/gtk/fm-standard-view.c
+++ b/src/gtk/fm-standard-view.c
@@ -687,7 +687,6 @@ static inline void create_list_view(FmStandardView* fv, GList* sels)
                                         FM_FOLDER_MODEL_COL_NAME);
     }
 
-    gtk_tree_view_set_reorderable(GTK_TREE_VIEW(fv->view), TRUE);
     gtk_tree_view_set_rules_hint(GTK_TREE_VIEW(fv->view), TRUE);
     gtk_tree_view_set_rubber_banding(GTK_TREE_VIEW(fv->view), TRUE);
     exo_tree_view_set_single_click((ExoTreeView*)fv->view, fm_config->single_click);
-- 
1.8.0.1

